### PR TITLE
Characters DNA Designs

### PR DESCRIPTION
**Pull request by SeanBerget**
## Changelog:

- Added DNA for the following characters:
    - Alonsus Faol _(by **SeanBerget**)_
    - Tandred Proudmoore _(by @icelille999)_
    - ~~Jaina Proudmoore _(by @icelille999)_~~
    - Jaina Proudmoore _(by @ercarp)_
    - Frost King Malakk _(by @ercarp)_
    - Grand Ma'da Ateena _(by @ercarp)_
    - Alleria Windrunner _(by **Herbe**)_

- Remade Jaina's DNA:

<details><summary>Click to expand</summary>

image
image

</details>

## To-do:
- [x] Remake Jaina's DNA to match her BfA appearance more.

# How to test:
- Review the added DNA in-game _(some characters may not be available in every bookmark)_ and give your feedback.
  - See if Jaina's new DNA is better than the old.

### CODE REVIEW

**Comment by robmart** [comment]
Would recommend changing the changelog to some thing like  

Added/changed DNA to the following characters:
* Saidan Dathrohan
* Taelan Fordring
* Liam Greymane
* Alonsus Faol
* Aedalas Blackmoore
* Grommash Hellscream

That way it is easier to understand what was changed at a glance

File: common/dna_data/wc_human_dna.txt [hunk 1]
```diff
@@ -2699,4 +2699,608 @@ aiden_perenolde_dna = { # by ercarp
...
+ 		gene_bs_ear_tip={ "ear_tip_pos" 0 "ear_tip_pos" 0 }
+ 		gene_bs_fingers={ "fingers_troll" 0 "fingers_troll" 0 }
+ 		gene_skin_value={ "skin_value_pos" 47 "skin_value_neg" 0 }
+ 		gene_race={ "creature_human" 113 "creature_human" 113 }
+        }
+    }
+	enabled=yes
+}
+
+liam_greymane_dna = {
```
Comment (zumbak04) [inline common/dna_data/wc_human_dna.txt:2824]: Liam has hair color closer to red.
<details>
<summary>Click to expand</summary>

image

</details>

Plus, this hairstyle is too bushy.

File: common/dna_data/wc_human_dna.txt [hunk 2]
```diff
@@ -2699,4 +2699,608 @@ aiden_perenolde_dna = { # by ercarp
...
+ 		gene_bs_ear_tip={ "ear_tip_pos" 0 "ear_tip_pos" 0 }
+ 		gene_bs_fingers={ "fingers_troll" 0 "fingers_troll" 127 }
+ 		gene_skin_value={ "skin_value_pos" 0 "skin_value_neg" 0 }
+ 		gene_race={ "creature_human" 141 "creature_human" 141 }vCX XC
+		}
+    }
+	enabled=yes
+}
+
+aedalas_blackmoore_dna = {
```
Comment (zumbak04) [inline common/dna_data/wc_human_dna.txt:3066]: What you used for a reference? He looks too chubby.
<details>
<summary>Click to expand</summary>

image

</details>

File: common/dna_data/wc_human_dna.txt [hunk 3]
```diff
@@ -2699,4 +2699,608 @@ aiden_perenolde_dna = { # by ercarp
...
+ 		gene_bs_ear_tip={ "ear_tip_pos" 0 "ear_tip_pos" 0 }
+ 		gene_bs_fingers={ "fingers_troll" 0 "fingers_troll" 0 }
+ 		gene_skin_value={ "skin_value_pos" 0 "skin_value_neg" 0 }
+ 		gene_race={ "creature_human" 61 "creature_human" 61 }
+        }
+    }
+	enabled=yes
+}
+
+alonsus_faol_dna = {
```
Comment (zumbak04) [inline common/dna_data/wc_human_dna.txt:3187]: What did you use for a reference? On my arts he has a pointy nose and doesn't have chubby cheeks.
<details>
<summary>Click to expand</summary>

image

</details>

File: common/dna_data/wc_orc_dna.txt [hunk 4]
```diff
@@ -1893,3 +1893,124 @@ griselda_dna = { # by icelille999
 	}
 	enabled=yes
 }
+
+garrosh_hellscream_dna = {
```
Comment (zumbak04) [inline common/dna_data/wc_orc_dna.txt:1897]: You create some visible clipping here.
<details>
<summary>Click to expand</summary>

image

</details>

Comment (zumbak04) [reply common/dna_data/wc_orc_dna.txt:1897]: I would work a bit more on Garrosh.
Change his eyes to red (orange).
Work on his cheeks.
<details>
<summary>Click to expand</summary>

image

</details>

File: common/dna_data/wc_human_dna.txt [hunk 5]
```diff
@@ -2699,4 +2699,608 @@ aiden_perenolde_dna = { # by ercarp
...
+ 		gene_bs_ear_tip={ "ear_tip_pos" 0 "ear_tip_pos" 0 }
+ 		gene_bs_fingers={ "fingers_troll" 0 "fingers_troll" 0 }
+ 		gene_skin_value={ "skin_value_pos" 0 "skin_value_neg" 0 }
+ 		gene_race={ "creature_human" 240 "creature_human" 240 }
+		}
+	}
+	enabled=yes
+}
+
+taelan_fordring_dna = {
```
Comment (zumbak04) [inline common/dna_data/wc_human_dna.txt:2945]: He has red hairs? What did you use for a reference?
<details>
<summary>Click to expand</summary>

image

</details>

**Review by zumbak04 — CHANGES_REQUESTED** [review]

**Comment by zumbak04** [comment]
> WIP
> 
> * Danath Trollbane
> * Galen Trollbane
> * Darius Crowley
> * Thargas Anvilmar

WIP? Better put them on a new branch.

**Comment by zumbak04** [comment]
> * There is no hair added for Aedalas Blackmoore or Alonsus Faol

Wait, why?

**Review by SeanBerget — COMMENTED** [review]
alright ill fix that tonight, also how do i see which beard and hair scripts belong to what type of beard/hair in the ruler designer as they have different names from ingame than in the folders.
like in folders "Big with fancy mustache" would be named "long wavy beard"..

**Comment by zumbak04** [comment]
> like in folders "Big with fancy mustache" would be named "long wavy beard"..

If fixed this if you find more, tell me.

File: common/dna_data/wc_human_dna.txt (hunk 1 above)
Comment (SeanBerget) [reply common/dna_data/wc_human_dna.txt:2824]: liam is blonde 100%

File: common/dna_data/wc_human_dna.txt [hunk 6]
```diff
@@ -2699,4 +2699,608 @@ aiden_perenolde_dna = { # by ercarp
 		}
 	}
 	enabled=yes
-}
\ No newline at end of file
+}
+
+saidan_dathrohan_dna = {
```
[the remark this answers is not in the document]
Comment (SeanBerget) [reply common/dna_data/wc_human_dna.txt:2704]: i know i havent added hair to ihm

File: common/dna_data/wc_human_dna.txt (hunk 2 above)
Comment (SeanBerget) [reply common/dna_data/wc_human_dna.txt:3066]: tru

File: common/dna_data/wc_human_dna.txt (hunk 3 above)
Comment (SeanBerget) [reply common/dna_data/wc_human_dna.txt:3187]: it is said in wowpedia that he is similar to santa clause, short and stout

Comment (SeanBerget) [reply common/dna_data/wc_human_dna.txt:3187]: i made him more slim, i havent just updated these files

File: common/dna_data/wc_human_dna.txt (hunk 5 above)
Comment (SeanBerget) [reply common/dna_data/wc_human_dna.txt:2945]: he has 3 different colors in his picture, brown, blonde and red hair which should i use

**Review by SeanBerget — COMMENTED** [review]
merge plez

File: common/dna_data/wc_human_dna.txt (hunk 2 above)
Comment (ercarp) [reply common/dna_data/wc_human_dna.txt]: DNA will be reworked on the `dna-2` branch.

62dd73f68a2e1f45710f41f5537d3395321dd192

File: common/dna_data/wc_human_dna.txt (hunk 1 above)
Comment (ercarp) [reply common/dna_data/wc_human_dna.txt]: DNA will be reworked on the `dna-2` branch.

62dd73f68a2e1f45710f41f5537d3395321dd192

File: common/dna_data/wc_human_dna.txt (hunk 6 above)
[the remark this answers is not in the document]
Comment (ercarp) [reply common/dna_data/wc_human_dna.txt]: DNA will be reworked on the `dna-2` branch.

62dd73f68a2e1f45710f41f5537d3395321dd192

File: common/dna_data/wc_human_dna.txt (hunk 5 above)
Comment (ercarp) [reply common/dna_data/wc_human_dna.txt]: DNA will be reworked on the `dna-2` branch.

62dd73f68a2e1f45710f41f5537d3395321dd192

**Review by Remerod — COMMENTED** [review]
My take on these Characters; 
Alonsus Faol, looks good, perhaps his hari/beard could be a little more yellowish/brownish and less bleach blonde? and does he not have brown eyes in wow? Otherwise he looks good.

<details>
<summary>Click to expand</summary>

image

image

image

</details>

Frost King Malakk; looks deffinitly ok, his wow model have pupils though and ours don't having new tusk that's more similar tot the wow model would help, but is not a must in my opinion. 

<details>
<summary>Click to expand</summary> 

image

image

</details>

Grand Ma'da Ateena; looks good, we dont really know how she looks i think but it would be nice if she could have tied shorter hair
<details>
<summary>Click to expand</summary> 

image

image

</details>
Alleria Windrunner; I don't really have anything to add here, it works, it looks good, and i can see that it is Alleria.
<details>
<summary>Click to expand</summary> 

image

</details>

Tandred Proudmoore; Tandred looks _really_ good. I do feel his eyes are a bit tealish blue while his WoW model is more purpleish blue.
<details>
<summary>Click to expand</summary> 

image

image

</details>
Jaina Proudmoore; Jaina looks like a beliable jaina, but at the same time it could be anyone else. Jaina have had a lot of looks, she is probably a very hard character to nail right, I think the nose, mouth and maybe yes can be approved. But again since she such an iconic character and been done in so many different art and models it might be hard to get it "right". So my take is that she works now but could maybe be improved if you wanna research a lot of Jaina art/models ¯\_(ツ)_/¯

<details>
<summary>Click to expand</summary> 

image

image

image

image

image

</details>

**Comment by zumbak04** [comment]
> Jaina Proudmoore; Jaina looks like a beliable jaina, but at the same time it could be anyone else. Jaina have had a lot of looks, she is probably a very hard character to nail right, I think the nose, mouth and maybe yes can be approved. But again since she such an iconic character and been done in so many different art and models it might be hard to get it "right". So my take is that she works now but could maybe be improved if you wanna research a lot of Jaina art/models ¯\_(ツ)_/¯

I agree Jaina should be changed to her look from the BfA.

**Comment by ercarp** [comment]
> 
> 
> > Jaina Proudmoore; Jaina looks like a beliable jaina, but at the same time it could be anyone else. Jaina have had a lot of looks, she is probably a very hard character to nail right, I think the nose, mouth and maybe yes can be approved. But again since she such an iconic character and been done in so many different art and models it might be hard to get it "right". So my take is that she works now but could maybe be improved if you wanna research a lot of Jaina art/models ¯_(ツ)_/¯
> 
> I agree Jaina should be changed to her look from the BfA.

Gotcha. I'll remake Jaina's DNA.

**Comment by ercarp** [comment]
@zumbak04 @Remerod Remade Jaina's DNA in 500702538320916d6874c8e0038b8f5f2663ae1a.

**Review by Remerod — COMMENTED** [review]
I think Jaina looks better now, I still don't really feel that "Iconic" look that she has, but again that might be impossible to achieve since she have had many different look as i stated before. "If" i could suggest a slight change then perhaps the lips/mouth and perhaps(or not) the chin? And i think here eyes are to "tealish" blue, i think they should be slightly dark or more light blue perhaps.

<details>
<summary>Click to expand</summary>

image

image


image

</details>

And maybe a slightly lighter skin tone? but she have more of the ck3 skin tone in much art, so it might just be fine as it is.

But over all, i don't think there "have" to be a change, she looks believable.

**Comment by zumbak04** [comment]
> "If" i could suggest a slight change then perhaps the lips/mouth and perhaps(or not) the chin?

I would agree on her chin/jaw. If it's possible, I would make her jaw a little less rounded.

Also, try to make her nose smaller?

<details>
<summary>Click to expand</summary>

image
image

</details>

**Comment by ercarp** [comment]
Applied your suggestions to Jaina in 2bf3177c5953136a1753423053c3dbc55910e498.

**Review by zumbak04 — APPROVED** [review]
LGTM! Good job!

**Comment by Remerod** [comment]
Jaina looks probably the best we can make her, so good job!
Unless you wanna do anymore changes to any other character i'll go ahead and give my approval once i'll get an awnser

**Review by Remerod — APPROVED** [review]
The Character designs seem's good to me, great work.

**Review by ercarp — APPROVED** [review]

**Pull request merged**